public/app/plugins/datasource/grafana-azure-monitor-datasource/azure_monitor/azure_monitor_datasource.ts

```diff
@@ -162,8 +159,8 @@ export default class AzureMonitorDatasource extends DataSourceWithBackend<AzureM
       })
       .then((result) => {
         return filter(result, (t) => {
```

Comment:
maybe while you're on it can replace this weird lodash filter with `result.filter(r => supportedMetricNamespaces.some( ..))`

Reply:
:+1: 

Note for myself: this function `getMetricDefinitions` is no longer useful since we are not using definitions anymore, we should probably remove it eventually.